packages/cardano-services/src/Program/utils.ts

```diff
@@ -55,6 +55,19 @@ export const createDnsResolver = (config: RetryBackoffConfig, logger: Logger) =>
 
 export type DnsResolver = ReturnType<typeof createDnsResolver>;
 
+export const connectionErrorCodes = ['ENOTFOUND', 'ECONNREFUSED', 'ECONNRESET'];
```

Comment:
Since the repeated logic is used across two different packages, where we prefer to export the common function, into `util` or `util-dev` package maybe ?

Reply:
`util` :+1:

Any package named `*-dev` is a `devDependency` BTW, so that wouldn't be an option here.

Reply:
629e967ee41670107a9f2fb26e0328366fadfa51

Reply:
How about adding a `network` directory to set some context?

`./src/network/isConnectionError.ts`